app/bundles/ConfigBundle/Event/ConfigBuilderEvent.php

```diff
@@ -115,37 +115,6 @@ public function getFormThemes()
         return $this->formThemes;
     }
 
-    /**
-     * Helper method can load $parameters array from a config file.
-     *
-     * @param string $path (relative from the root dir)
-     *
-     * @return array
-     */
-    public function getParameters($path = null)
-    {
-        $paramsFile = $this->pathsHelper->getSystemPath('app').$path;
-
-        if (file_exists($paramsFile)) {
-            // Import the bundle configuration, $parameters is defined in this file
-            include $paramsFile;
-        }
-
-        if (!isset($parameters)) {
-            $parameters = [];
-        }
-
-        $fields     = $this->getBase64EncodedFields();
```

Comment:
This method does not exist in the whole Mautic codebase. That's why I suggest to remove the getParameters method because no one noticed except PHPSTAN.